### PR TITLE
add assignments

### PR DESCRIPTION
**Pull request by Sarojpoudeler**

### CODE REVIEW

**Comment by prertik** [comment]
https://github.com/Nischalkhadka/Quality-Assurance/pull/7#issuecomment-573582889

**Comment by Sarojpoudeler** [comment]
commit message updated

**Comment by Nischalkhadka** [comment]
Conflicts. Do resolve them first.

**Comment by Sarojpoudeler** [comment]
conflict resolved. Please check it once.

File: QA Interns 2020/Assignments/participants.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Add your name in an orderly fashion.
+
 1. Sheshraj Kandel
+2. Saroj Poudel
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/participants.md]: Just add your name only.

File: QA Interns 2020/Assignments/saroj/JSON/Answer.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+"errorCode":"VALIDATION_ERROR_INVOICE_SUPPLIER_TAX_FEWER_THAN_TOTAL_OFLINE_ITEM_SUPPLIER_TAXES",
+"groupId":"null",
+"messageKey":"null",
+"message":"Invoice Supplier Tax is less than the total of all Invoice line items Supplier Taxes",
+"messageArguments":"null",
+"context":{ 
+
+}
+}
+]
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/saroj/JSON/Answer.json]: Add a line space

File: QA Interns 2020/Assignments/saroj/JSON/question.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+"errorCode":"VALIDATION_ERROR_INVOICE_SUPPLIER_TAX_FEWER_THAN_TOTAL_OFLINE_ITEM_SUPPLIER_TAXES",
+"groupId":null,
+"messageKey":null,
+"message":"Invoice Supplier Tax is less than the total of all Invoice line items Supplier Taxes",
+"messageArguments":null,
+"context":{ 
+
+}
+}
+]
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/saroj/JSON/question.json]: Add a line space

File: QA Interns 2020/Assignments/saroj/PostmanEnvironment/TestEnvironment.postman_environment.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+	"id": "49bcda78-2fe0-46b1-9f37-13873ec57197",
+	"name": "TestEnvironment",
+	"values": [],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2020-01-16T08:24:08.518Z",
+	"_postman_exported_using": "Postman/7.14.0"
+}
```
Comment (Nischalkhadka) [inline QA Interns 2020/Assignments/saroj/PostmanEnvironment/TestEnvironment.postman_environment.json]: Add a line space

**Comment by Sarojpoudeler** [comment]
All conflicts are resolved as mentioned by you  please review it

**Comment by Nischalkhadka** [comment]
Are you sure that you have resolved the conflicts? Do review once again and try to resolve the conflicts.

**Pull request merged**